core/modules/node/node.types.inc

```diff
@@ -181,16 +181,18 @@ function node_type_form($form, &$form_state, $type = NULL) {
   );
   $form['workflow']['status_default'] = array(
     '#type' => 'radios',
-    '#title' => t('Default status'),
+    '#title' => t('Default publish action'),
     '#options' => array(
-      NODE_PUBLISHED => t('Published'),
-      NODE_NOT_PUBLISHED => t('Draft'),
+      NODE_PUBLISHED => t('Publish immediately'),
+      NODE_NOT_PUBLISHED => t('Save as draft'),
       NODE_SCHEDULED => t('Schedule for later'),
     ),
     '#default_value' => $type->settings['status_default'],
-    '#description' => t('Content saved as draft is only accessible to the content creator or site administrators.') . ' ' . t('Content set to "Schedule for later" will be saved as draft when the content is created, but will be automatically published on the date and time selected.'),
     '#required' => TRUE,
   );
+  $form['workflow']['status_default'][NODE_PUBLISHED]['#description'] = t('Published content can be accessed by all site visitors.');
```

Comment:
Remove the word "all"

Reply:
Looking at the PR sandbox, "All" is removed from the content type settings but not from the node form.